library/build.gradle

```diff
@@ -35,6 +35,7 @@ android {
 dependencies {
     androidTestCompile 'com.android.support.test:runner:0.5'
     androidTestCompile 'com.android.support.test:rules:0.5'
+    compile 'com.android.support:support-annotations:25.3.1'
```

Comment:
I am personally very much in support of adding nullability annotations. Useful for us Java users, too!